src/uwtools/cli.py

```diff
@@ -492,7 +492,7 @@ def _add_subparser_mpas_init(subparsers: Subparsers) -> ModeChecks:
 
     :param subparsers: Parent parser's subparsers, to add this subparser to.
     """
-    parser = _add_subparser(subparsers, STR.mpasinit, "Execute MPASInit tasks")
+    parser = _add_subparser(subparsers, STR.mpasinit, "Execute MPAS Init tasks")
```

Comment:
`MPASInit` looked a bit to me like an internal detail (the name of `class MPASInit`) leaking out and less like a description for human users.